core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/LruBlockCache.java

```diff
@@ -762,7 +762,7 @@ public Lock getLoadLock(String blockName) {
       return null;
 
     // Would rather use Guava Striped, but its @Beta
-    int index = Math.abs(blockName.hashCode()) % loadLocks.length;
+    int index = Math.abs(blockName.hashCode() % loadLocks.length) ;
```

Comment:
curious why this was a spotbug issue.... but either way should work.

Reply:
The error code was RV_ABSOLUTE_VALUE_OF_HASHCODE

Below is one post discussing it.  There are two integers that `Math.abs(blockName.hashCode())` maps to the same thing.

https://stackoverflow.com/questions/23416264/bad-attempt-to-compute-absolute-value

Reply:
Should use `Math.floorMod(blockName.hashCode(), loadLocks.length)` instead.